tests/unit/s2n_change_cipher_spec_test.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include "tls/s2n_connection.h"
+#include "tls/s2n_crypto.h"
+#include "tls/s2n_tls.h"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    uint8_t empty_finished_array[S2N_SSL_FINISHED_LEN] = { 0 };
```

Comment:
the tests are setting a 1.2 cipher suite, I think we want `S2N_TLS_FINISHED_LEN`